app.js

```diff
@@ -2,20 +2,24 @@ const express = require("express");
 const {
   getCategories,
   getReviewById,
+  patchReviews,
 } = require("./controllers/categories_controller");
 
```

Comment:
Here you are requiring all controllers from one file that should, by naming convention only contain controllers for requests relating to categories. 

This will be the same in your model files. Take some time to create review specific model and controller files before moving on. This will keep your code better encapsulated.